lib/components/app/locale-selector.tsx

```diff
@@ -44,31 +44,42 @@ const LocaleSelector = (props: LocaleSelectorProps): JSX.Element => {
   }
 
   return (
-    <NavDropdown
+    <Dropdown
       id="locale-selector"
-      pullRight
+      // TODO: How to make this work without block ruby?
+      style={{ display: 'block ruby' }}
       title={
-        <StyledIconWrapper style={{ color: 'rgba(255, 255, 255, 0.85)' }}>
+        <span
+          style={{
+            color: 'rgba(255, 255, 255, 0.85)'
+          }}
+        >
           <GlobeAmericas />
-        </StyledIconWrapper>
+        </span>
       }
     >
-      {Object.keys(configLanguages)
-        .filter((locale) => locale !== 'allLanguages')
-        .map((locale) => (
-          <MenuItem
-            className="locale-name"
-            key={locale}
-            onClick={(e: MouseEvent) => handleLocaleSelection(e, locale)}
-          >
-            <span
-              style={locale === currentLocale ? { fontWeight: 'bold' } : {}}
+      <ul style={{ listStyle: 'none', margin: 0, padding: 0 }}>
+        {Object.keys(configLanguages)
+          .filter((locale) => locale !== 'allLanguages')
+          .map((locale) => (
+            <li
+              aria-selected={locale === currentLocale}
+              key={locale}
+              onClick={(e: MouseEvent) => handleLocaleSelection(e, locale)}
+              onKeyPress={(e: any) => handleLocaleSelection(e, locale)}
+              // We are correct, not eslint: https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html
+              // eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role
+              role="option"
             >
-              {configLanguages[locale].name}
-            </span>
-          </MenuItem>
-        ))}
-    </NavDropdown>
+              <span
```

Comment:
This needs to be a `<button>` instead, and the `onClick` handler above goes there (remove the `onKeyPress` handler.

Reply:
I originally had a button, but it caused too many conflicts with the bootstrap nav bar styles. Will add a TODO.

Reply:
The subitems are not accessible by keyboard otherwise.

Reply:
Let me see if I can get it working...

Reply:
Latest changes are better. Did you want to address the following items in this PR or have me handle it separately?
- Implement ESC to dismiss the menu when the menu is open
- Using up/down keys to highlight the previous/next menu item (was available in the bootstrap dropdown)
- If focus "escapes" from the menu, don't show the menu (or alternately, keep the focus in the menu while it is open). With bootstrap, while a dropdown is open, you could tab to the item next to the dropdown.

Reply:
> If focus "escapes" from the menu, don't show the menu

This one still needs to be resolved.

Reply:
Addressed with 1dbf5552071e5040daccba1d8b30981b94b4776d